src/commands/openYAMLConfigFile.ts

```diff
@@ -27,11 +28,29 @@ export async function openYAMLConfigFile(context: IActionContext, node?: StaticW
             const ymlFsPath: string = path.join(fsPath, ymlFileName);
             // if we couldn't find it, then try opening it in GitHub
             if (await fse.pathExists(ymlFsPath)) {
-                await window.showTextDocument(await workspace.openTextDocument(ymlFsPath));
+                const configDocument: TextDocument = await workspace.openTextDocument(ymlFsPath);
+                const selection: Range | undefined = await getSelection(configDocument, buildConfigToSelect);
+                await window.showTextDocument(configDocument, { selection });
                 return;
             }
         }
     }
 
     await openUrl(`${node.repositoryUrl}/edit/${node.branch}/${ymlFileName}`);
 }
+
+async function getSelection(configDocument: TextDocument, buildConfigToSelect?: BuildConfig): Promise<Range | undefined> {
```

Comment:
It looks like that package is used for parsing specific values out of yaml files but since we're looking for the buildConfig's location in the document I think regex is the way to go